src/app/mork/api/v1/routers/tasks.py

```diff
@@ -32,7 +32,7 @@ async def create_task(
 ) -> TaskResponse:
     """Create a new task."""
     celery_task = TASK_TYPE_TO_FUNC[task.type]
-    celery_params = task.model_dump(exclude="type")
+    celery_params = task.model_dump(exclude="type", exclude_none=True)
```

Comment:
Just for my understanding, why do you need to do that ?

Reply:
Here we are passing the parameter from the `task` pydantic model (which is built from the request) to the Celery task.

When the parameter `limit` is not set in the HTTP request, the field is still included in the model dump as such:
```
{
    "limit": None,
}
```
As I want to rely on the default limit parameter of the task function (`limit = 0`), I am excluding "none" fields from the dump